### PR TITLE
Skip and log invalid docs

### PR DESCRIPTION
**Pull request by kwahlin**

### CODE REVIEW

**Review by olovy — APPROVED** [review]

File: whelktool/README.md [hunk 1]
```diff
@@ -22,20 +22,30 @@ Options
...
+                                    (defaults to "reports").
+ -s,--step                          Change one document at a time,
+                                    prompting to continue.
+ -T,--no-threads                    Do not use threads to parallellize
+                                    batch processing.
+ -t,--num-threads <N>               Override default number of threads
+                                    (32).
+ -v,--validation <MODE>             [UNSAFE] Set JSON-LD validation mode.
+                                    Defaults to ON. Possible values:
+                                    ON/OFF/LOG-ONLY
```
Comment (olovy) [inline whelktool/README.md:48]: It's LOG_ONLY

File: whelktool/src/main/groovy/whelk/datatool/WhelkTool.groovy [hunk 2]
```diff
@@ -819,6 +832,15 @@ class WhelkTool {
         }
     }
 
+    private static ValidationMode parseValidationMode(String arg) {
```
Comment (olovy) [inline whelktool/src/main/groovy/whelk/datatool/WhelkTool.groovy:835]: It should fail on an invalid option instead of silently defaulting

**Review by olovy — CHANGES_REQUESTED** [review]
See comments, other than that LGTM!

File: whelktool/src/main/groovy/whelk/datatool/WhelkTool.groovy (hunk 2 above)
Comment (kwahlin) [reply whelktool/src/main/groovy/whelk/datatool/WhelkTool.groovy]: [c1eb8ae](https://github.com/libris/librisxl/pull/1518/commits/c1eb8ae77a2197970d3c432b6a3371fbb0e6b9e5)

**Review by olovy — APPROVED** [review]

**Pull request merged**